stable/sparkoperator/values.yaml

```diff
@@ -4,8 +4,20 @@ imagePullPolicy: IfNotPresent
 imagePullSecrets: []
 replicas: 1
 
+deployment:
+  create: false
```

Comment:
Shouldn't be true by default?
The values.yaml usually contain defaults s.t. there's some usability in just doing a simple helm install of the chart and getting some version of it working